rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -0,0 +1,40 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.Address;
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.NetworkParameters;
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.bitcoinj.script.ScriptBuilder;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.ethereum.crypto.HashUtil;
+
+public class BitcoinTestUtils {
+
+    public static List<BtcECKey> getBtcEcKeysFromSeeds(String[] seeds, boolean sorted) {
+        List<BtcECKey> keys = Arrays
+            .stream(seeds)
+            .map(seed -> BtcECKey.fromPrivate(HashUtil.keccak256(seed.getBytes(StandardCharsets.UTF_8))))
+            .collect(Collectors.toList());
+
+        if (sorted) {
```

Comment:
If **sorted** Is True, Why Do We Still Go Ahead And Sort It?
Maybe The Right Variable Would Be **sort**

Reply:
The method is called `getBtcEcKeysFromSeeds`, so it reads getBtcEcKeysFromSeeds sorted, that's the way I see it at least